std/file.d

```diff
@@ -2983,6 +2983,31 @@ auto dirEntries(string path, SpanMode mode, bool followSymlink = true)
     return DirIterator(path, mode, followSymlink);
 }
 
+/// Duplicate functionality of D1's $(D std.file.listdir()):
+unittest
+{
+  string[] listdir(string pathname)
```

Comment:
Doesn't ddoc strip initial indentation from code samples? Or is that still not enough?

Also, if we're going to do this for this code example, shouldn't we also do it for all other Phobos code examples?